### ISSUE TITLE
`Load More Stack Frames` renders poorly when frame names are long

### ISSUE THREAD

**Issue by HolyBlackCat** [microsoft/vscode#225594]
Does this issue occur when all extensions are disabled?: N/A (need *some* debugger extension to test)

- VS Code Version: 1.92.1
- OS Version: Arch LInux x64 6.6.44-3-lts

Steps to Reproduce:

1. Open the debugger for any language.
2. Pause on a breakpoint. Either with a deep enough stack so that it doesn't load fully and shows `Show N More Stack Frames` message. OR some debugger extensions apparently don't load the stack at all by default, and display `Load More Stack Frames` after the first entry.
3. Make sure the stack frames already shown have very long names.
4. Observe that the `Load More Stack Frames` (or `Show N More Stack Frames`) is centered relative to the longest frame name, making it completely invisible if frame names are long and the panel is narrow. See picture:

Completely invisible on small panel widths:
![pic](https://github.com/user-attachments/assets/c57c1294-8211-4514-b1af-8ba057f56b8a)

Only starts appearing on large panel widths:
![pic](https://github.com/user-attachments/assets/49bec37b-1ab3-4976-9bae-1dc9e4f9c180)

I suggest that this text should be centered based on the physical panel width (ignoring the scroll).

**Comment by roblourens** [comment]
Is this with `workbench.list.horizontalScrolling` enabled? Or no?

**Comment by HolyBlackCat** [comment]
Yep! Doesn't happen with `"workbench.list.horizontalScrolling": false`.

**Comment by HolyBlackCat** [comment]
@roblourens I have provided the information requested, friendly ping to update the issue status.
